main.go

```diff
@@ -95,6 +96,9 @@ func main() {
 	flag.IntVar(&machineDeploymentConcurrency, "machinedeployment-concurrency", 10,
 		"Number of machine deployments to process simultaneously")
 
+	flag.BoolVar(&kubeadmBootstrapperEnabled, "enable-kubeadm-bootstrapper", true,
```

Comment:
Boolean flags that default to `true` are a bit weird to deal with (having to set `-enable-kubeadm-bootstrapper=false`), instead should the flag be `disable-kubeadm-bootstrapper` instead?

Reply:
fixed in amended commit